hazelcast-jet-distribution/src/main/resources/release_notes.txt

```diff
@@ -5,6 +5,47 @@ issues at https://github.com/hazelcast/hazelcast-jet/issues/<issue number>.
 
 The label in the brackets refer to the respective module.
 
+** Hazelcast Jet 3.1 **
+
+1. New Features
+
+[pipeline-api] Enable composite transforms using `apply` operator [#1352]
+[pipeline-api] Add support for fault tolerant sources to SourceBuilder API [#1394]
+[core] YAML configuration support [#1371]
+[integration] Source and sink for Elasticsearch [https://github.com/hazelcast/hazelcast-jet-contrib/tree/master/influxdb]
+[integration] Source and sink for InfluxDB [https://github.com/hazelcast/hazelcast-jet-contrib/tree/master/elasticsearch]
```

Comment:
I think the links need to be fixed for the above two entries.

Reply:
how do you mean, fixed?

Reply:
for Elasticsearch the link reads https://github.com/hazelcast/hazelcast-jet-contrib/tree/master/influxdb

and for the Influxdb it reads https://github.com/hazelcast/hazelcast-jet-contrib/tree/master/elasticsearch

they should be replaced with each other?

Reply:
Thanks, I didn't notice! fixed